hangrybot.py

```diff
@@ -88,6 +95,36 @@ def remind_to_go(self, channel="#lunch"):
             text=text
         )
 
+    def gorgonzola(self):
+        # get daily menus
+        bc = BackmarktCrawler()
+        menu_bc = bc.get_menus()
+        cc = CoronaCrawler()
+        menu_cc = cc.get_menus()
+
+        # search for gorgonzola
+        text = ''
+        if "gorgonzola" in menu_bc.lower() and "gorgonzola" in menu_cc.lower():
+            text = 'It has to be either Corona or Backmarkt!:star-struck:'
+
+        elif "gorgonzola" in menu_cc.lower():
+            text = 'I think @darioxz wants to go to Corona...:nauseated_face:'
+
+        elif "gorgonzola" in menu_bc.lower():
+            text = 'Apparently, @darioxz wants to go to Backmarkt...:unamused:'
+
+        return text
+
+    def dario(self, channel):
+        text = self.gorgonzola(self)
+        if text == '':
+            text = "I don't know, there is no Gorgonzola...:sob:"
```

Comment:
I would add this case as the `else` case above. That way the logic is on the same level.

Reply:
I didn't do that because `gorgonzola()` is being used for the daily menu as well and I don't want an output for menus without Gorgonzola.